### PR TITLE
Enhancement/ui cleanup

### PR DESCRIPTION
**Pull request by rjgrazioli**
Just polished up a few things to more closely resemble the latest designs. I'll probably hit up UI next to adjust sidebar widths and stuff.

### CODE REVIEW

File: src/components/admin-integrations/index.tsx [hunk 1]
```diff
@@ -32,6 +32,8 @@ import collectionServiceAuthorizationCreate from '../../actions/collection/servi
 import collectionServiceAuthorizationUpdate from '../../actions/collection/service-authorizations/update';
 import collectionServiceAuthorizationDestroy from '../../actions/collection/service-authorizations/destroy';
 
+const DASHBOARD_BACKGROUND = '#FAFAFA';
```
Comment (1egoman) [inline src/components/admin-integrations/index.tsx]: I'd prefer this to be defined in one place and imported in versus in each file that it's used. Thoughts on that?

Comment (rjgrazioli) [reply src/components/admin-integrations/index.tsx]: yeah, for sure. we should just be able to reference the variables in UI, right?

Comment (rjgrazioli) [reply src/components/admin-integrations/index.tsx]: `import colorVariables from '@density/ui/variables/colors.json'`

Comment (guscost) [reply src/components/admin-integrations/index.tsx]: yes

**Pull request merged**